backend/package.json

```diff
@@ -12,7 +12,9 @@
     "@types/socket.io": "^2.1.4",
     "body-parser": "^1.19.0",
     "express": "^4.17.1",
-    "socket.io": "^2.3.0",
+    "socket.io": "^2.3.0"
+  },
+  "devDependencies": {
     "typescript": "^3.6.4"
```

Comment:
Is there a reason for doing this? It would be better to have everything under dependencies.